interface/new/new_comprehensive.php

```diff
@@ -416,24 +416,25 @@ function searchme() {
 <form action='new_comprehensive_save.php' name='demographics_form' id="DEM"  method='post' onsubmit='return submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,"DEM",constraints)'>
 
     <div class="container-fluid">
-        <div class="row">
-            <div class="col-xs-12">
-                <div class="page-header">
-                    <h1><?php echo xlt('Search or Add Patient');?></h1>
+        <div class="page-header">
```

Comment:
Since we're working on the code in this PR and it's relavent still suggest we do it now